app/database/queries/breeze_buddy/human_assist.py

```
@@ -0,0 +1,930 @@
+    owner_clause = f"AND {record}->>'opened_by' = $2" if opened_by else ""
+    values: List[Any] = [conversation_id]
+    if opened_by:
+        values.append(opened_by)
+    query = f"""
+        UPDATE chat_session cs
+        SET last_activity_at = now()
+        WHERE cs.id = $1::uuid
+          AND {record}->>'status' = 'OPEN'
+          {owner_clause}
+        RETURNING {_columns("cs")}
+    """
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)